opennebula/resource_opennebula_template_test.go

```diff
@@ -233,7 +273,7 @@ resource "opennebula_template" "template" {
 
   context = {
     dns_hostname = "yes"
-    network = "YES"
+    NETWORK = "YES"
```

Comment:
Why did you change the case ? is it for test purposes ?
Not a big deal but it's just add a bit of noise for this PR.

Reply:
In the file `opennebula/resource_opennebula_virtual_machine_test.go`, there are a lot of resources with the key `context`. All these context-definitions have uppercase-keys, so it was easier to update this file `opennebula/resource_opennebula_template_test.go`. This is also an example of conflicts between uppercase-text (Opennebula) and lowercase-text (Terraform).